### DISCUSSION TITLE
can't find /shared/app dir in v2.1.0, that was in v2.0.5

### DISCUSSION THREAD

**Discussion by Elvis-ch1** [argoproj/argo-cd#7055] [General]
Hello firstly thank you very much for this very useful tool. v2.0.5 and lower had a /shared/app dir, please where has that been moved to in v2.1.0. We attempted upgrading to v2.1.0 and it failed because it could not find the /shared/app directory.
please help thank you.

**Comment by alexmt** [comment]
This is a regression. Thanks for reporting @Elvis-ch1!

I've created issue to track the fix to support custom styling feature: https://github.com/argoproj/argo-cd/issues/7065


Can you please provide a little more details about the failure?  Want to make sure I'm not missing something else

**Reply by Elvis-ch1** [reply]
@alexmt thank you for looking into that. 
Do you know how soon, there will be a new release that will include this fix?
